Lib/DocXMLRPCServer.py

```diff
@@ -20,6 +20,17 @@
             CGIXMLRPCRequestHandler,
             resolve_dotted_attribute)
 
+
+def _html_escape_quote(s, quote=True):
```

Comment:
Oh, my suggestion was to remove the quote parameter since it's always quote in this code. Either rename the function to _html_escape() and keep the parameter, or keep this name but remove the parameter.